catalog/ui/catalog-ui-search/src/main/webapp/js/ApplicationStart.js

```diff
@@ -26,17 +26,38 @@ require([
 
     var workspaces = store.get('workspaces');
 
+    function getWorkspacesOwnedByUser() {
+        return workspaces.filter(workspace => user.isGuest() ? workspace.get('localStorage') === true : workspace.get('metacard.owner') === user.get('user').get('email'));
```

Comment:
✏️ just a little nit you can ignore if you don't care, but this line is quite long and could very well benefit from that bountiful vertical screen real estate that is so easily scrollable. It might even, perhaps, add just a small token of readability to this line of code.